src/Core/Resources/ErrorMessages.resx

```diff
@@ -141,10 +141,6 @@
     <value>Value cannot be null</value>
     <comment>Do not translate: the text of an internal error.</comment>
   </data>
-  <data name="ValueCannotBeEmptyOrWhiteSpace" xml:space="preserve">
```

Comment:
@sfoslund I'm assuming that this has no impact on the loc process, but can you confirm?

Reply:
No, I don't believe so.